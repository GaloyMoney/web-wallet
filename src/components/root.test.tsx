import "@testing-library/jest-dom/extend-expect"

import appRoutes from "../server/routes"
import { render } from "@testing-library/react"

import Root from "./root"

describe("Root", () => {
  it("renders Home and matches snapshot", () => {
    const { asFragment } = render(<Root initialData={{ appRoutes, path: "/" }} />)
    expect(asFragment()).toMatchSnapshot()
  })
  it("renders Login and matches snapshot", () => {
    const { asFragment } = render(<Root initialData={{ appRoutes, path: "/login" }} />)
    expect(asFragment()).toMatchSnapshot()
  })
})
