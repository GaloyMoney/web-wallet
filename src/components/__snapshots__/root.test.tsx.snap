// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root renders Home and matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="home"
  >
    <div
      class="header"
    >
      <div
        class="balance"
      >
        <div
          class="title"
        >
          Current Balance
        </div>
        <div
          class="value"
        >
          <div
            class="primary"
          >
            $0
          </div>
          <div
            class="secondary"
          >
            (0 sats)
          </div>
        </div>
      </div>
      <a
        href="/login"
      >
        Login
      </a>
    </div>
  </div>
  <div
    class="footer"
  >
    <div
      class="powered-by"
    >
      Powerd By 
      <a
        href="https://galoy.io/"
        rel="noreferrer"
        target="_blank"
      >
        Galoy
      </a>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Root renders Login and matches snapshot 1`] = `
<DocumentFragment>
  <form>
    <input
      type="tel"
    />
  </form>
  <div
    class="footer"
  >
    <div
      class="powered-by"
    >
      Powerd By 
      <a
        href="https://galoy.io/"
        rel="noreferrer"
        target="_blank"
      >
        Galoy
      </a>
    </div>
  </div>
</DocumentFragment>
`;
